free-3.md

```diff
@@ -57,12 +75,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think that a decent amount of the project would involve translating between the various social media bot API services and the GUI of this language. That could involve a direction translation with many similar functions or it could involve clever software design and a lot of pattern matching. Depending on the implementation, the time allocations for the project could vary. The similarities between different social media APIs would determine how much of the project would involve writing functions specific to one type of bot.
```

Comment:
Could you perhaps elaborate more on the language aspects of the project? I couldn't really tell which parts would be considered language. To me, it seems like what you listed are all the systems aspect. What kind of language design decisions do you think you'd need to make?